src/utils/sort.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"sort"
+	"strconv"
+)
+
+// SortData helps sort table rows. It sorts the table based on values given in the sortIdx column and sorts ascending if sortAsc is true. sortCase is set to identify the set of 'less' functions to use to sort the selected column by.
+func SortData(data [][]string, sortIdx int, sortAsc bool, sortCase string) {
+
+	// Define less functions
+	intSort := func(i, j int) bool {
+		x, _ := strconv.Atoi(data[i][sortIdx])
+		y, _ := strconv.Atoi(data[j][sortIdx])
+		if sortAsc {
+			return x < y
+		}
+		return x > y
+	}
+
+	strSort := func(i, j int) bool {
+		if sortAsc {
+			return data[i][sortIdx] < data[j][sortIdx]
+		}
+		return data[i][sortIdx] > data[j][sortIdx]
+	}
+
+	floatSort := func(i, j int) bool {
+		x1 := data[i][sortIdx]
+		y1 := data[j][sortIdx]
+		x, _ := strconv.ParseFloat(x1[:len(x1)-1], 32)
```

Comment:
Why the `len - 1`? Won't it mess up the conversion? 

Ref: https://play.golang.org/p/nr1JvhpmS-d

Reply:
To remove the % symbol